src/main/java/junit/framework/JUnit4TestAdapterCache.java

```diff
@@ -29,7 +29,7 @@ public Test asTest(Description description) {
         }
     }
 
-    Test createTest(Description description) {
+    private Test createTest(Description description) {
```

Comment:
Sorry, typo. I think it is **not** worth the risk. We should  not change the modifiers of this field.

I don't think we should ever reduce the visibility of a field or method in a public class (unless we have not released a version of JUnit where that member existed)